canonical-websites/www.ubuntu.com
# Removed non-Vanilla matrix

## Done

Removed non-Vanilla matrix from u.c and updated two existing pages with it

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/internet-of-things/robotics](http://0.0.0.0:8001/internet-of-things/robotics) and [/openstack/storage](http://0.0.0.0:8001/openstack/storage)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the pages look ok

## Issue / Card

Fixes #4568 

## Screenshots

robotics
![image](https://user-images.githubusercontent.com/441217/51482355-e9704c80-1d9e-11e9-8a05-2444b570e2bb.png)

storage
![image](https://user-images.githubusercontent.com/441217/51482377-f55c0e80-1d9e-11e9-8692-522fa919648f.png)
